aurora/hardhat.config.js

```diff
@@ -101,14 +101,25 @@ task('unlock', 'Unlock tokens on Near')
         "If the CONFIG_NAME is provided, the config with path ./configs/CONFIG_NAME.json will be used.")
     .addParam('fastBridgeAddress', 'Aurora Fast Bridge address')
     .addParam('nonce', 'Nonce of the Fast Bridge transfer')
-    .addParam('ethTokenAddress', "Token address on Eth")
-    .addParam('validTillBlockHeight', "Valid till block height")
     .setAction(async taskArgs => {
         const { unlock } = require('./scripts/utils');
         const [signer] = await hre.ethers.getSigners();
         const config = require(`./configs/${taskArgs.auroraFastBridgeConfigName}.json`);
 
-        await unlock(signer, config, taskArgs.fastBridgeAddress, taskArgs.nonce, taskArgs.ethTokenAddress, taskArgs.validTillBlockHeight);
+        await unlock(signer, config, taskArgs.fastBridgeAddress, taskArgs.nonce);
+    });
+
+task('get_pending_transfers', 'Unlock tokens on Near')
```

Comment:
```suggestion
task('get_pending_transfer', 'Get pending transfer by nonce on Near')
```
This command returns just one transfer by nonce

Reply:
fixed: https://github.com/aurora-is-near/fast-bridge-protocol/pull/130/commits/7556ca83c35179d100ede6c99eb21bfd252f6403